### ISSUE TITLE
readme.md improvement suggestion. 

### ISSUE THREAD

**Issue by wheelsailor** [woocommerce/wc-api-php#74]
Hi
May I suggest the following improvement for the description in the readme.md: 
A PHP wrapper for the WooCommerce REST API. Easily interact with the WooCommerce REST API *securely* using this library. *If using a HTTPS connection this library uses BasicAuth, else it uses Oauth to provide a secure connection to WooCommerce*

As there was nothing here to tell me this - in the end I looked at the code. I assumed it was only Using BasicAuth (because code is set out like BasicAuth - nothing like the Oauth signature base string code on the woo-commerce API docs) so with my website not being HTTPS, it assumed it would be insecure - because it doesn't tell me it is... 

Good news is that I got it working too easily and then got over worried about Oauth.

**Comment by claudiosanches** [comment]
It's not even possible to perform Basic Auth with your store is not under HTTPS.
On the REST API docs have more details about it: https://woocommerce.github.io/woocommerce-rest-api-docs/

**Comment by wheelsailor** [comment]
Hi Claudio
I spent quite a lot of time reading the API docs you linked yesterday before posting this, especially the Introduction and Authentication sections. 
While this is an "official library" I still think that confirmation of how the security is working behind the scenes is useful to users, especially new users starting out. It would have saved me 6 hours yesterday. 

I can't see where it explicitly states its not possible to perform Basic Auth without HTTPS. (I do appreciate  this is a good thing in hindsight). 
From the docs you linked (Which I have spent hours looking at):

_You may access the API over either HTTP or HTTPS, but HTTPS is recommended where possible._
OK.
No where in the section under "Authentication/Authentication over HTTPS" does it say it won't work using HTTP. 

So to try it out and my first plugin, I used the example code from HTTPS Basic Auth section of the documentation (I wasn't worried about it not being HTTPS connection as I am in a local development wordpress install and I would change the keys before deployment). It worked which was nice, so then I proceeded to the HTTP OAuth section as I knew my connection was not HTTPS. At that point I got very lost, even with google's help. 

_You must use OAuth 1.0a “one-legged” authentication to ensure REST API credentials cannot be intercepted by an attacker._ Very understandable.  To have _"The official libraries default to OAuth if you are not using HTTPS"_ would have been nice here. 
So I read that and was concerned how this library had implemented security. I was surprised it just worked, which made me assume it was using Basic Auth, considering the long instructions on how to create a signature base string which just seemed to not get back to the same point as the Basic Auth example. 

While I appreciate the desire to keep the docs as succinct as possible, on a topic as critical as security, I believe that a few more comments is adding value, and gives users confidence to proceed, knowing that they are not stuffing up the security of the shop. 

Regards
Nick
